.travis.yml

```diff
@@ -33,6 +34,24 @@ jobs:
       after_script:
         - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
   
+    - name: "regression test using python-behave thread 1"
```

Comment:
I don't really like these names. `thread 1` is not really meaningfull,

Can we add better names? What is this thread for?